Corona-Warn-App/src/main/res/values-de/legal_strings.xml

```diff
@@ -244,9 +244,9 @@
     <!-- XTXT: Submission consent privacy card subtitle -->
     <string name="srs_privacy_card_subtitle_text" translatable="false">Durch Antippen von „Einverstanden“ willigen Sie in folgende Schritte ein:</string>
     <!-- XTXT: Submission consent privacy card first bulletpoint -->
-    <string name="srs_privacy_card_first_bulletpoint_text" translatable="false">Die App teilt Ihr positives Testergebnis, um Nutzer, denen Sie begegnet sind, zu warnen. Dies betrifft auch Nutzer der Corona-Warn-App, die zeitgleich am selben Event oder Ort wie Sie eingecheckt waren.</string>
+    <string name="srs_privacy_card_first_bulletpoint_text" translatable="false"><b>Die App teilt Ihr positives Testergebnis, um andere Nutzer der Corona-Warn-App zu warnen, denen Sie begegnet sind oder die zeitgleich am selben Event oder Ort wie Sie eingecheckt waren.</b></string>
     <!-- XTXT: Submission consent privacy card second bulletpoint -->
-    <string name="srs_privacy_card_second_bulletpoint_text" translatable="false">Ihre Angaben zur Art des Tests werden anonym statistisch ausgewertet.</string>
+    <string name="srs_privacy_card_second_bulletpoint_text" translatable="false">Bevor das Testergebnis geteilt werden kann, wird die <b>Echtheit Ihrer App einmalig geprüft</b>. Dazu wird durch Ihr Smartphone eine eindeutige Kennung erzeugt und an Google in die USA oder andere Drittländer übermittelt, damit Google die Echtheit Ihrer App gegenüber dem RKI bestätigen kann. Die Kennung enthält Informationen über die Version Ihres Smartphones und der App. Google kann damit möglicherweise auf Ihre Identität schließen und nachvollziehen, dass die Echtheitsprüfung Ihres Smartphones stattgefunden hat. Weitere Angaben aus der App erhält Google hierbei nicht.</string>
```

Comment:
I just noticed that this section doesn't have all the text in bold like the other ones. Right now it's all bold for us. Can you check with the designers if that's what they want?

Reply:
Yes, it seems its just the section of the second bullet point displayed bold (See Figma design)
<img width="196" alt="image" src="https://user-images.githubusercontent.com/22813752/203354722-e9c1d592-c786-4b9c-86d5-b187e5c2e4e3.png">